stream-chat-android-ui-components/src/main/res/layout/stream_ui_message_preview_item.xml

```diff
@@ -18,8 +18,6 @@
         app:layout_constraintBottom_toBottomOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent"
-        app:streamAvatarHeight="@dimen/stream_ui_avatar_size_medium"
-        app:streamAvatarWidth="@dimen/stream_ui_avatar_size_medium"
```

Comment:
These are not parsed for the new `AvatarView`, since https://github.com/GetStream/stream-chat-android/pull/908